File: PYTHON/Send.py
import pika

connection = pika.BlockingConnection(pika.ConnectionParameters('localhost'))
channel = connection.channel()

channel.queue_declare(queue='hello')

message = 'publicador python conectado...'

channel.basic_publish(exchange='', routing_key='hello', body=message)
print(f" Mensaje enviado: {message}")

while message != "exit":
    message = input()
    channel.basic_publish(exchange='', routing_key='hello', body=message)
    print(f" Mensaje enviado: {message}")

File: C#/Send/Send.cs
using System;
using System.Text;
using RabbitMQ.Client;

namespace Send 
{
    class Program
    {
        static void Main(string[] args)
        {
            var factory = new ConnectionFactory() {HostName = "localhost"};
            using(var connection = factory.CreateConnection())
            {
                using(var channel = connection.CreateModel())
                {
                    channel.QueueDeclare(queue: "hello", durable: false, exclusive:false, autoDelete:false, arguments: null);

                    string message = "Servicio iniciado...";
                    var body = Encoding.UTF8.GetBytes(message);
                    channel.BasicPublish(exchange: "", routingKey: "hello", basicProperties: null, body: body);
                    while (message != "exit") {
                        message = Console.ReadLine();
                        body = Encoding.UTF8.GetBytes(message);
                        channel.BasicPublish(exchange: "", routingKey: "hello", basicProperties: null, body: body);
                        Console.WriteLine($"Mensaje enviado: {message}");
                    }
                }
            }
            Console.WriteLine("Oprima cualquier tecla para salir");
            Console.ReadLine();
        }
    }
}